app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,9 @@
+class IngredientsController < ApplicationController
+  def show
+    @ingredient = Ingredient.find(params[:id])
+  end
+
+  def new
```

Comment:
```suggestion
  def index
```
Update `new` action to `index` action